lms/djangoapps/mobile_api/video_outlines/tests.py

```diff
@@ -201,12 +272,80 @@ def test_section_under_course(self):
         unit_url = course_outline[0]["unit_url"]
         self.assertRegexpMatches(
             section_url,
-            r'courseware/test_factory_section_omega_%CE%A9/$'
+            r'courseware/test_factory_section_under_course_omega_%CE%A9/$'
         )
-        self.assertTrue(section_url)
-        self.assertTrue(unit_url)
+
         self.assertEqual(section_url, unit_url)
 
+        self._verify_paths(
+            course_outline,
+            [
+                u'test factory section under course omega \u03a9',
+            ]
+        )
+
+    def test_structure_course_section_video(self):
```

Comment:
Isn't this the same name as on L259?
Should this be renamed to `test_structure_course_chap_section_video`?